packages/shared/src/lib/core/token/enums/token-standard.enum.ts

```diff
@@ -3,5 +3,4 @@ export enum TokenStandard {
     Irc27 = 'IRC27',
```

Comment:
is there a reason this one is still in here?

Reply:
I just didn't wanna mess with it, but if it's barely used I will just replace it.